qiskit/circuit/quantumcircuit.py

```diff
@@ -63,7 +74,7 @@ def __init__(self, *regs, name=None):
             # pylint: disable=not-callable
             # (known pylint bug: https://github.com/PyCQA/pylint/issues/1699)
             if sys.platform != "win32" and \
-               isinstance(mp.current_process(), mp.context.ForkProcess):
+                    isinstance(mp.current_process(), mp.context.ForkProcess):
```

Comment:
It's pycharm that have a strong opinion about correct indentation...  I put it everything in the same line because it fits and have my IDE happy in b29464c9